cmd/mt-index-cat/out/out.go

```diff
@@ -7,33 +7,32 @@ import (
 	"text/template"
 
 	"github.com/davecgh/go-spew/spew"
-
-	"github.com/raintank/schema"
+	"github.com/grafana/metrictank/idx"
 )
 
 var QueryTime int64
 
-func Dump(d schema.MetricDefinition) {
+func Dump(d idx.MetricDefinition) {
```

Comment:
what happens if you spew.Dump a idx.MetricDefinition ? it's probably not going to be very readable, containing uintptr's and all that i presume. it would be nice if we had a method to dump out the structure in the same format as before

Reply:
Yes I agree, I will make that happen.